torch/_inductor/codecache.py

```diff
@@ -57,6 +58,36 @@ def cache_dir():
     )
 
 
+class DiskCache:
+    @staticmethod
+    @functools.lru_cache(None)
+    def _subdir():
+        subdir = os.path.join(cache_dir(), "cached_tunings")
+        os.makedirs(subdir, exist_ok=True)
+        return subdir
```

Comment:
Did this throw on internal deployments? Or just work as is?

Reply:
It worked fine on a devserver.  This writes to `/tmp`